lib/src/cli.dart

```diff
@@ -15,16 +15,19 @@ import 'singleton.dart' as singleton;
 // This version must be updated in tandem with the pubspec version.
 const String appVersion = '0.8.5';
 
-List<String> grinderArgs() => _args;
-List<String> _args;
-bool _verifyProjectRoot;
+List<String> grinderArgs() {
+  var args = _args;
+  if (args == null) fail('grinderArgs() may only be called after grind().');
```

Comment:
Instead of assigning to a local we could also check `_args` and after that return `_args!`. That would be slightly more terse.